packages/integration-tests/tests/retrospective-demo/step2-group.test.ts

```diff
@@ -116,13 +111,6 @@ test.describe('retrospective-demo / group page', () => {
     const documentingInNotionCard = page.locator('text=Documenting things in notion')
     await dragReflectionCard(decisionsInOneOnOnesCard, documentingInNotionCard)
 
-    // Then it auto-generates a header
-    await expect(
-      page.locator(
-        `[data-cy=group-column-Start] [data-cy*="Start-group-"] input[value="Things Notion"]`
```

Comment:
Now that we're using AI to generate group titles, they could be anything, so I've removed these tests